lib/profile_events_queue_handler.ts

```diff
@@ -49,33 +49,48 @@ const publicApiKey = getRequiredStringEnv("PUBLIC_API_KEY");
 
 type WelcomeMessages = ReadonlyArray<(p: ExtendedProfile) => NewMessage>;
 
-// TODO: decide text for welcome message
+// TODO: internal links
 // TODO: switch text based on user's preferred_language
 const welcomeMessages: WelcomeMessages = [
   (profile: ExtendedProfile) =>
     NewMessage.decode({
       content: {
-        markdown: `# Hello new user ${profile.email || ""}
-
-  We welcome you to the Digital Citizenship API program  
-  This is a welcome message to test if the system works.`,
-
-        subject: `Welcome new user ${profile.email || ""}`
+        markdown: `**Benvenuto su IO, l'applicazione dei servizi pubblici a disposizione di tutti i cittadini italiani !**
+
+Scopri le funzioni e impara a usare l'app del cittadino.
+IO ti consente di ricevere messaggi dalle Pubbliche Amministrazioni italiane, sia locali che nazionali e all'occorrenza effettuare pagamenti.
+Puoi decidere da chi e come essere contattato, dalla sezione [preferenze](ioit://PREFERENCES_HOME) di questa applicazione.
+Per esempio puoi decidere di ricevere i messaggi anche sulla tua e-mail associata a SPID${
+          // tslint:disable-next-line:restrict-plus-operands
+          profile.email ? ":" + profile.email : ""
+        }.
+Se hai già usato pagoPA per effettuare pagamenti verso la Pubblica Amministrazione, potrai vedere lo storico delle transazioni
+ed eventuali carte di credito salvate nella sezione [portafoglio](ioit://WALLET_HOME).
+Altrimenti, sempre dal [portafoglio](ioit://WALLET_HOME) puoi aggiungere i tuoi metodi di pagamento preferiti, oppure pagare direttamente
+un avviso pagoPA leggendo il QR code di un avviso cartaceo.
+Se qualcosa non ti dovesse essere chiaro durante l'utilizzo dell'app, clicca il punto di domanda che trovi in alto a destra.`,
+
+        subject: `Benvenuto ! Scopri le funzioni principali e impara a usare l'app dei servizi pubblici.`
```

Comment:
@matteodesanti  subject looks a little long, also it looks like there's an extra space after _Benvenuto_.

What about: _Benvenuto su IO, l'app dei servizi pubblici!_